packages/circuit-ui/tsconfig.json

```diff
@@ -45,5 +43,5 @@
     "**/*.spec.*",
     "**/*.stories.*"
   ],
-  "typeRoots": ["./node_modules/@types", "./types/"]
```

Comment:
I tried to remove the `@types` from our type roots to see if they were what was causing the es2018 lib to be used by the compiler. Apparently not. But I don't think we want to do this—do we? It seems like we might inadvertently include types for libraries that we're not using (dependencies' dependencies)

Reply:
If this doesn’t break type resolution for dependencies, then I don’t see a problem in removing it.